### PR TITLE
Updated HOF

### LINKED ISSUE #10: update hof in readme

### PR DESCRIPTION
**Pull request by PoseidonSigmaX**
Fixes #10 
-
Updated Hall of Fame to include a second contributor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,4 +81,4 @@ Read our contributing [guide](CONTRIBUTING.md) to get started with contributing
 
 Thank you to all the contributors who have helped us in making this project better 🙌
 
-<a href="https://github.com/arshadkazmi42"><img src="https://github.com/arshadkazmi42.png" width="30" /></a>
+<a href="https://github.com/arshadkazmi42"><img src="https://github.com/arshadkazmi42.png" width="30" /></a>  <a href="https://github.com/wdmcquinn"><img src="https://github.com/wdmcquinn.png" width="30" /></a>
```
Comment (arshadkazmi42) [inline README.md]: `/></a>  <a href`

can you remove space here.
we would not want to show spaces

Comment (PoseidonSigmaX) [reply README.md]: Okay sure

**Comment by PoseidonSigmaX** [comment]
Alright, I've made the changes :)

**Comment by arshadkazmi42** [comment]
Thank you for working on this.

**Comment by PoseidonSigmaX** [comment]
> Thank you for working on this.

Thank you for the opportunity! These first few successful steps encourage me to do bigger things :)

**Pull request merged**